GsaGH/Parameters/5_Results/GsaNodeDisplacement.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GsaGH.Parameters._5_Results {
+  public class GsaNodeDisplacement {
```

Comment:
```suggestion
  public class GsaNodeDisplacements {
```
File name should be changed as well